scripts/test.sh

```diff
@@ -20,6 +20,7 @@ cd packages/sample-babel7      && yarn test
 cd packages/sample-typescript2 && yarn test
 cd packages/sample-noconflict  && yarn test
 cd packages/sample-noparse     && yarn test
-cd packages/ember-auto-import  && yarn test
+# There are no tests in here and it's throwing an error anyway
```

Comment:
I couldn't reproduce this error, and would prefer not to disable this test suite, because it creates a trap for a future developer who tries to put tests in the standard place for an addon and then they don't run.

Reply:
I do occasionally get EENT errors about existing tmp directories in the packages, but I was able to get it to pass by cleaning, re-yarn'ing

Reply:
I suspect something within broccoli is sharing the same temp directories between runs and may not be completely safe to parallelize. I haven't been able to narrow down exactly what.